src/SOFe/Capital/Transfer/Config.php

```diff
@@ -33,91 +34,101 @@ public function __construct(
     public static function parse(Parser $config, Context $di, Raw $raw) : Generator {
         false && yield;
 
-        return new self(
-            transferMethods: [
-                new CommandMethod(
-                    command: "pay",
-                    permission: "capital.transfer.pay",
-                    defaultOpOnly: false,
-                    src: new ParameterizedLabelSelector([
-                        AccountLabels::PLAYER_UUID => "{sender uuid}",
-                        Constants::LABEL_CURRENCY => Constants::CURRENCY_NAME,
-                    ]),
-                    dest: new ParameterizedLabelSelector([
-                        AccountLabels::PLAYER_UUID => "{recipient uuid}",
-                        Constants::LABEL_CURRENCY => Constants::CURRENCY_NAME,
-                    ]),
-                    rate: 1.0,
-                    minimumAmount: 0,
-                    maximumAmount: 10000,
-                    transactionLabels: new ParameterizedLabelSet([
-                        Constants::LABEL_PAYMENT => "",
-                    ]),
-                    messages: new Messages(
-                        notifySenderSuccess: '{green}You have sent ${sentAmount} to ${recipient}. You now have ${srcBalance} left.',
-                        notifyRecipientSuccess: '{green}You have received ${receivedAmount} from ${sender}. You now have ${destBalance} left.',
-                        noSourceAccounts: '{red}There are no accounts to send money from.',
-                        noDestinationAccounts: '{red}There are no accounts to send money to.',
-                        underflow: '{red}You do not have ${sentAmount}.',
-                        overflow: '{red}The accounts of {recipient} are full. They cannot fit in ${sentAmount} more.',
-                        internalError: '{red}An internal error occurred. Please try again.',
-                    ),
+        $transferParser = $config->enter("transfer", <<<'EOT'
+            "transfer" tells Capital what methods admins and players can send money through.
+            EOT);
+
+        $commandsParser = $transferParser->enter("commands", <<<'EOT'
+            These slash commands initiate transfers.
```

Comment:
There are no non-slash commands in PocketMine; users won't confuse this with Discord commands after all.

```suggestion
            These commands initiate transfers.
```

Reply:
Fixed in ce88f7e78023ffe09939780d008bc6aa2d9f38f1.